functions/venue.js

```diff
@@ -68,6 +68,54 @@ const PlacementState = {
   Hidden: "HIDDEN",
```

Comment:
[note] The changes in this file _should_ just be moving the position of the functions around, there shouldn't be any changes to the code within them.